hawc/apps/summary/models.py

```diff
@@ -149,6 +151,54 @@ def print_node(node, depth):
                 print_node(node, 2)
 
 
+class SummaryTable(models.Model):
+    objects = managers.SummaryTableManager()
+
+    DUMMY = 0
+    EVIDENCE_PROFILE = 1
+    EVIDENCE_INTEGRATION = 2
+
+    TABLE_CHOICES = (
+        (DUMMY, "Dummy table"),
+        (EVIDENCE_PROFILE, "Evidence profile table"),
+        (EVIDENCE_INTEGRATION, "Evidence integration table"),
+    )
+
+    assessment = models.ForeignKey(Assessment, on_delete=models.CASCADE)
+    title = models.CharField(max_length=128)
+    slug = models.SlugField(
+        verbose_name="URL Name",
+        help_text="The URL (web address) used to describe this object "
+        "(no spaces or special-characters).",
+    )
+    content = models.JSONField(default=dict)
+    table_type = models.PositiveSmallIntegerField(choices=TABLE_CHOICES, default=DUMMY)
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+    def get_assessment(self):
+        return self.assessment
+
+    def to_docx(self):
+        table = GenericTable.parse_obj(self.content)
+        return table.to_docx()
+
+    def clean(self):
+        # content validation
+        try:
+            if self.table_type == self.DUMMY:
```

Comment:
pull this if statement out into a `get_content_schema_class` method which returns a pydantic class